5_databases/week1/lesson1(HW).sql

```diff
@@ -0,0 +1,43 @@
+-- Active: 1699540080106@@0.0.0.0@3306@hyf_lesson1
+
+SELECT COUNT(*)
+FROM task;
+
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+select task.title
+FROM task
+WHERE status_id = 3;
+
+SELECT task.title
+FROM task
+WHERE status_id != 3;
+
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+SELECT task.title, task.due_date
+FROM task
+WHERE title LIKE "%database%";
+
+SELECT task.title, status.name
```

Comment:
This returns the task 3 times with all statuses which means 103 rows of tasks, but we only have 35 tasks. We only want 1 task and its current status:

SELECT task.title, status.name
FROM task
JOIN status
ON status.id=task.status_id;